roles/oneview_id_pools_ipv4_range_facts/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+# defaults file for oneview_id_pools_ipv4_ranges
+config: "~/.ansible/collections/ansible_collections/hpe/oneview/roles/oneview_id_pools_ipv4_ranges/files/oneview_config.json"
```

Comment:
change the path to config file. Replace "oneview_id_pools_ipv4_ranges" with "oneview_id_pools_ipv4_range_facts"

Reply:
fixed